aws/values-amazon-secondary.yaml

```diff
@@ -21,5 +22,17 @@ prometheus:
     global:
       external_labels:
         cluster_id: kubecostProductConfigs_clusterName
+    retention: 2d
+  # nodeExporter:
```

Comment:
Per our discussion, if we are to leave the option to disable node exporter, let's include the service account piece as well.
https://guide.kubecost.com/hc/en-us/articles/4407601830679-Troubleshoot-Install#a-name-node-exporter-a-issue-failedscheduling-kubecost-prometheus-node-exporter